src/sonic-config-engine/minigraph.py

```diff
@@ -446,6 +533,22 @@ def parse_meta(meta, hname):
                     region = value
     return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id, region
 
+def parse_asic_meta(meta, hname):
+    deployment_id = None
+    sub_role = None
+    device_metas = meta.find(str(QName(ns, "Devices")))
+    for device in device_metas.findall(str(QName(ns1, "DeviceMetadata"))):
+        if device.find(str(QName(ns1, "Name"))).text.lower() == hname.lower():
+            properties = device.find(str(QName(ns1, "Properties")))
+            for device_property in properties.findall(str(QName(ns1, "DeviceProperty"))):
+                name = device_property.find(str(QName(ns1, "Name"))).text
+                value = device_property.find(str(QName(ns1, "Value"))).text
+                if name == "DeploymentId":
+                    deployment_id = value
```

Comment:
what is the use of deployment id in the asic meta?

Reply:
not used, removed it from parse_asic_meta